_includes/ziele.html

```diff
@@ -1,9 +1,9 @@
 <section id="ziele">
-    <h2 class="section-heading">Was wir machen</h2>
+    <h2 class="section-heading">Was uns Vereine verbindet</h2>
     <hr class="primary">
-    <div class="flex-grid">
+    <div class="row">
     {% for ziel in site.data.ziele %}
-        <div class="flex-cell text-center">
+        <div class="col-lg-4 col-12 text-center">
             <div class="service-box">
                 <i class="fa fa-4x fa-{{ ziel.fontawesome-icon }} wow bounceIn text-primary"></i>
```

Comment:
```css
.text-primary {
	color: #007bff !important;
}
```
The primary-color should be changed to `#83aa16` (TechAachen Green).

Reply:
Das hatte ich nicht gefunden, wo man das ändern kann.

Reply:
Hmmm. Ich suche mal...

Ich befürchte, dass Bootstrap das irgendwo setzt/überschreibt.

Reply:
Das Problem löst sich wahrswcheinlich von selbst, wenn wir das alte Bootstrap 3 entfernen udn das CSS mal aufräumen.

Reply:
Working since Bootstrap 3 was removed.